code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2084,4 +2084,171 @@ All effects don't start immediately, but rather get worse over time; the rate is
+
+/datum/reagent/consumable/ethanol/catsip/on_mob_life(mob/living/M)
+	if(prob(8))
+		M.say(pick("Nya.", "N-nya!", "NYA!"), forced = "catsip")
+	return ..()
+
+/datum/reagent/consumable/ethanol/catsip/on_mob_add(mob/living/carbon/human/M)
+	if(!M.dna.species.is_wagging_tail())
+		M.emote("wag")
+	return ..()
+
+/datum/reagent/consumable/ethanol/rubberneck
+	name = "Rubberneck"
+	id = "rubberneck"
+	description = "A quality rubberneck should not contain any gross natural ingredients."
+	color = "#ffe65b"
+	boozepwr = 60
+	quality = DRINK_GOOD
+	taste_description = "artifical fruityness"
+	glass_icon_state = "rubberneck"
+	glass_name = "Rubberneck"
+	glass_desc = "A popular drink amongst those adhering to an all synthetic diet."
+
+/datum/reagent/consumable/ethanol/duplex
+	name = "Duplex"
+	id = "duplex"
+	description = "An insperable combination of two fruity drinks."
+	color = "#50e5cf"
+	boozepwr = 25
+	quality = DRINK_NICE
+	taste_description = "green apples and blue raspberries"
+	glass_icon_state = "duplex"
+	glass_name = "Duplex"
+	glass_desc = "To imbibe one component separately from the other is consider a great faux pas."
+
+/datum/reagent/consumable/ethanol/trappist
+	name = "Trappist Beer"
+	id = "trappist"
+	description = "A strong dark ale brewed by space-monks."
+	color = "#390c00"
+	boozepwr = 40
+	quality = DRINK_VERYGOOD
+	taste_description = "dried plums and malt"
+	glass_icon_state = "trappistglass"
+	glass_name = "Trappist Beer"
+	glass_desc = "boozy Catholicism in a glass."
+
+/datum/reagent/consumable/ethanol/trappist/on_mob_life(mob/living/carbon/M)
+	if(!M.mind.isholy)
+		return ..()
+	M.adjustFireLoss(-2.5, 0)
+	M.jitteriness = max(0, M.jitteriness-1)
+	M.stuttering = max(0, M.stuttering-1)
+	return ..()
+
+/datum/reagent/consumable/ethanol/blazaam
+	name = "Blazaam"
+	id = "blazaam"
+	description = "A strange drink that few people seem to remember existing. Doubles as a Berenstain remover."
+	boozepwr = 70
+	quality = DRINK_FANTASTIC
+	taste_description = "alternate realities"
+	glass_icon_state = "blazaamglass"
+	glass_name = "Blazaam"
+	glass_desc = "The glass seems to be sliding between realities. Doubles as a Berenstain remover"
+	var/stored_teleports = 0
+	var/mandelamessage
+
+/datum/reagent/consumable/ethanol/blazaam/on_mob_life(mob/living/carbon/M)
+	if(M.drunkenness > 40)
+		if(stored_teleports)
+			do_teleport(M, get_turf(M), rand(1,3), channel = TELEPORT_CHANNEL_BLUESPACE)
+			stored_teleports--
+		if(prob(10))
+			stored_teleports += rand(2,6)
+			M.vomit()
+		if(prob(1))
+			mandelamessage = pick("Friend, we don't have much time, I need you to come rescue me.", "[M], i am still alive.", "Wake up. Please wake up.", "A winner is a dreamer who never gives up.", "[M], the fate of this world is in your hands.")
+			to_chat(M, "<span class='name'>Nelson Mandela </span><span class='message'>says, \"[mandelamessage]\"</span>")
```

Comment:
lol

Reply:
cringe

Reply:
I am not too attached to this part, i am willing to remove it.

Reply:
Removed this